### DISCUSSION TITLE
Can I use annual precipitation as input?

### DISCUSSION THREAD

**Discussion by bowenbelongstonature** [pism/pism#495] [Q&A] [answered]
If yes, what's the distribution among 12 months?

Or... PISM just allow the monthly precipitation.

**Comment by aaschwanden** [comment, marked as answer]
Yes, PISM can ready fields with different periodicities as long as the `time_bounds` are set correctly. See http://www.pism.io/docs/climate_forcing/time-dependent.html for details.
